cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -40,30 +40,48 @@
           "widget-attributes": {
             "values": [
               "auto-detect",
+              "africa-south1",
               "asia-east1",
               "asia-east2",
               "asia-northeast1",
               "asia-northeast2",
               "asia-northeast3",
               "asia-south1",
+              "asia-south2",
               "asia-southeast1",
               "asia-southeast2",
               "australia-southeast1",
+              "australia-southeast2",
+              "europe-central2",
               "europe-north1",
+              "europe-southwest1",
               "europe-west1",
+              "europe-west10",
+              "europe-west12",
               "europe-west2",
               "europe-west3",
               "europe-west4",
               "europe-west6",
+              "europe-west8",
+              "europe-west9",
+              "me-central1",
+              "me-central2",
+              "me-west1",
               "northamerica-northeast1",
+              "northamerica-northeast2",
               "southamerica-east1",
+              "southamerica-west1",
               "us-central1",
+              "us-central2",
               "us-east1",
               "us-east4",
+              "us-east5",
+              "us-south1",
               "us-west1",
               "us-west2",
               "us-west3",
-              "us-west4"
+              "us-west4",
+              "us-west8"
```

Comment:
Can you please confirm if Cloud Dataproc is supported in us-west8? https://screenshot.googleplex.com/7vaLoAXVGEAjFdf

Reply:
Yes, Dataproc is live in us-west8.

This region is shown in the Cloud Console.